### PR TITLE
Add documentation for __system_context__

### PR DESCRIPTION
**Pull request by joeldenning**
This PR documents https://github.com/webpack/webpack/pull/9119, which was released in https://github.com/webpack/webpack/releases/tag/v5.0.0-beta.15.

### CODE REVIEW

File: src/content/configuration/output.md [hunk 1]
```diff
@@ -862,8 +862,17 @@ System.register('my-library', [], function(_export) {
 });
 ```
 
-Module proof library.
```
Comment (joeldenning) [inline src/content/configuration/output.md:865]: This sentence seemed to be in error.

File: src/content/configuration/output.md [hunk 2]
```diff
@@ -862,8 +862,17 @@ System.register('my-library', [], function(_export) {
...
+As of webpack@5.0.0-beta.15, you can access [SystemJS context](https://github.com/systemjs/systemjs/blob/master/docs/system-register.md#format-definition) via `__system_context__`:
```
Comment (EugeneHlushko) [inline src/content/configuration/output.md:865]:
```suggestion
You can access [SystemJS context](https://github.com/systemjs/systemjs/blob/master/docs/system-register.md#format-definition) via `__system_context__`:
```

**Comment by joeldenning** [comment]
Thanks for the review - updated

**Comment by joeldenning** [comment]
Any update on this?

**Comment by EugeneHlushko** [comment]
Thank you @joeldenning 
Notifications went lose sorry for delay!

**Pull request merged**